rosboard/handlers.py

```diff
@@ -316,7 +316,7 @@ def check_origin(self, origin):
         """Allow connections from any origin (needed for Foxglove)"""
         return True
 
-    def initialize(self, node, max_allowed_latency, full_topics, allow_external_clients, google_auth_enabled, metrics_publisher=None, auto_shutdown_time=0):
+    def initialize(self, node, max_allowed_latency, full_topics, allow_external_clients, google_auth_enabled, metrics_publisher=None, auto_shutdown_time=0, front_end_auto_reconnect=True):
```

Comment:
```suggestion
    def initialize(self, node, max_allowed_latency, full_topics, allow_external_clients, google_auth_enabled, metrics_publisher=None, auto_shutdown_time=0, frontend_auto_reconnect=True):
```
frontend is all together